karate-core/src/main/java/com/intuit/karate/driver/DevToolsDriver.java

```diff
@@ -968,4 +969,29 @@ public void inputFile(String locator, String... relativePaths) {
         method("DOM.setFileInputFiles").param("files", files).param("nodeId", nodeId).send();
     }
 
+    public void enableAccessibility()
```

Comment:
attempted devtools protocols around - 

1. enable accessibility -> getFullAXTree ( this tree by itself isnt useful). Need some standard implementations to parse it.

2. enable audits -> this generates events around issues primarily CORS / mixed content type issues.

3. enable performance -> getMetrics gives current page load metrics.

All above not much useful ( except performance, which is already provided by lighthouse ).

Just wanted to demonstrate the attempt :(

Reply:
hmm can you point to a sample of `getFullAXTree` result if really needed, we can write a custom parser. I'm confident of doing these things if it is JSON. the new `rewrite` code can even handle scraping HTML BTW